packages/css/src/components/description-list/description-list.scss

```
@@ -0,0 +1,55 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+@import "../../common/text-rendering";
+
+@mixin reset {
+  box-sizing: border-box;
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+.ams-description-list {
+  color: var(--ams-description-list-color);
+  display: grid;
+  font-family: var(--ams-description-list-font-family);
+  font-size: var(--ams-description-list-font-size);
+  font-weight: var(--ams-description-list-font-weight);
+  gap: var(--ams-description-list-gap);
+  grid-template-columns: 1fr;
+  line-height: var(--ams-description-list-line-height);
+
+  @media screen and (min-width: $ams-breakpoint-medium) {
+    grid-template-columns: 1fr 2fr;
+  }
+
+  @include reset;
+}
+
+.ams-description-list--inverse-color {
+  color: var(--ams-description-list-inverse-color);
+}
+
+.ams-description-list__term {
+  @include reset;
+  @include text-rendering;
+}
```

Wouldn't adding the following rules here solve the whole multiple term, multiple details thing?

```suggestion
  @media screen and (min-width: $ams-breakpoint-medium) {
    grid-column-start: 1;
  }
}
```

---

No it does not, it will skip a row in the case of two `dt` elements followed by 3 `dd` elements. But I guess it's better then leaving it out. Perhaps should UX should describe how this case should look.

---

Terms in a column followed by definitions in a column is kind of how I would expect it to look. It's pretty close to the native styling as well. I'd go for it

---

It's in there now